Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM openjdk:8-jdk
 RUN apt-get update \
-    && apt-get -y install postgresql-client \
+    && apt-get -y install postgresql-client pkg-config \
```

Comment:
isn't this only necessary in the CI?

Reply:
Oops, sorry, I didn't mean to commit that.